lib/rails_critical_css_server/rewrite.rb

```diff
@@ -6,9 +6,8 @@ class Rewrite
 
     delegate :request, :params, to: :template
 
-    def initialize(template, manifest: template.asset_url('application.css'), version: Config.version, skip: false, cache: true, &block)
```

Comment:
The way this is written you should be able to override the name of the manifest manually when you call this method. I call it like this:

    = RailsCriticalCssServer::Rewrite.call(self,
      skip: current_user.present?,
      manifest: asset_url('css_manifest.css')) do